docs/whats_new/v1.rst

```diff
@@ -1,3 +1,17 @@
+Version 1.0.3
+=============
+
+Deployed: 17th April 2021
+
+Contributors
+____________
+    - Hector Patino
```

Comment:
Can it be that the underline needs to be ------- instead of the continuous line? rst is quite adamant about what you use to mark titles and subtitles @hectorpatino

Reply:
> 
> 
> @solegalli Ayudame Sole!!! Im not able to see what the error is accordding to circleci `/home/circleci/project/docs/whats_new/v1.rst:20:Title level inconsistent:` this is the error but I just don´t see it. What am I doing wrong?

Hice un PR a tu repo con algunos cambios :)